src/pybind/mgr/dashboard/frontend/src/app/ceph/pool/pool-details/pool-details.component.ts

```diff
@@ -61,12 +70,14 @@ export class PoolDetailsComponent implements OnChanges {
   ngOnChanges() {
     if (this.selection) {
       this.poolService.getConfiguration(this.selection.pool_name).subscribe((poolConf) => {
-        this.selectedPoolConfiguration = poolConf;
+        CdHelperClass.updateChanged(this, { selectedPoolConfiguration: poolConf });
+      });
+      CdHelperClass.updateChanged(this, {
+        poolDetails: _.omit(this.selection, this.omittedPoolAttributes),
+        hasTiers: !!(this.selection['tiers']?.length > 0),
+        isReplicated: this.selection['type'] === 'replicated',
+        grafanaPath: 'ceph-pool-detail?var-pool_name=' + this.selection['pool_name']
```

Comment:
> Rather than going on "hacking" the default Angular change detection by , why not (continue) moving to [a less sensitive change detection strategy](https://blog.ninja-squad.com/2018/09/27/angular-performances-part-4/)? And if we insist on this, let's [at least follow some guidelines](https://dev.to/nickraphael/ngonchanges-best-practice-always-use-simplechanges-always-1feg)

I think this applies to the datatable, too. because of that i opened https://github.com/ceph/ceph/pull/37044.

Reply:
It's not hacking angular it's just helping us to not assign to much or to carelessly ;) as Angulars change detection works pretty well, sometime to well for us as it seems :P